ledger/participant-integration-api/src/test/suite/scala/platform/indexer/RecoveringIndexerSpec.scala

```diff
@@ -131,7 +131,7 @@ class RecoveringIndexerSpec
         }
     }
 
-    "recover from failure" in newLoggingContext { implicit logCtx =>
+    "recover from failure" in newLoggingContext { implicit loggingContext =>
```

Comment:
See if it makes sense to use `LoggingContext.ForTesting` here and remove a level of nesting.